### PR TITLE
Added README

### PR DESCRIPTION
**Pull request by arshiagg**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Contiv Prometheus Integration for Kubernetes
+
```
Comment (neelimamukiri) [inline README.md]: Add a summary of what the exporter is and what it allows.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+## Getting Started
+
+In order to run this exporter for your cluster, please follow [this]() tutorial.
+
+## Exporter Architecture
+
+![Exporter Pipeline](exporter.png)
+
```
Comment (neelimamukiri) [inline README.md]: Can you split this up into sections:

1. Prometheus setup - configuration - yaml file that is volume mounted. Add a link to the file here and describe how the different sections allow it to do pod discovery.
2. Netplugin/Netmaster exporters - How the exporter is a web service providing a /metrics rest end point at different ports for master/netplugin
How the pods themselves are configured with annotations to allow for the data to be fetched.
3. Exporter implementation - Ruby code details
4. OVS details - how there is a server started by netplugin on the port that you access and how it has traffic stats
4. Data extraction pipeline - prometheus to exporter to netmaster/ovs pipeline

Comment (arshiagg) [inline README.md]: Neelima can you please expand on what information you're looking for point 4?

Comment (neelimamukiri) [inline README.md]: For 4. Netplugin uses OpenvSwitch to program the traffic flows. Netplugin starts an ovsdb-server at port 6640. ovsdb stores information about the traffic flows and statistics.
Exporter can access the ovsdb at localhost:6640 as it is running in the same network namespace, as part of the same pod.

**Pull request merged**